---
title: Dr. Rivera-Mariani Computational Approaches
---
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-103557590-4"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-103557590-4');
</script>

[**Home Page**](http://www.friveram.com/) | [**Credentials**](http://www.friveram.com/about) | [**Publications**](http://www.friveram.com/publications) | [**Projects**](http://www.friveram.com/projects) | [**Teaching**](http://www.friveram.com/teaching) 

## Computational Approaches

Find in this page information about Computational Approaches in Biology and Data Science employing R, Python, Matlab, including computational approaches...

- for big data analysis in biological sciences (computational biology)
- to monitor students' learning and performance in the classroom
- to implement machine learning methods

---

<div style="text-align:justify"><p><a href="https://github.com/friveramariani/Proteomic-Examples">Example of Proteomic Analysis of Ovarian Cancer</a>: This example proteomic data analysis, performed in the <a href="https://www.mathworks.com/products/matlab.html">Matlab</a> computer language, analyzed SELDI-TOF mass spectrometry from blood samples of two groups: ovarian cancer  vs control group. In this example, Matlab scripts are presented to pre-process the data, generate plots of the groups' spectrograms as well as ranking features using a two-way t-statistic. The data analyzed in this example was obtained from the <a href="https://home.ccr.cancer.gov/ncifdaproteomics/ppatterns.asp">FDA-NCI Clinical Proteomics Program</a>.</p></div> 

- [GitHub repository](https://github.com/friveramariani/Proteomic-Examples)
- [PDF output](https://www.researchgate.net/publication/319103946_Proteomics_data_analysis_in_cancer_biology_with_Matlab)

---

<div style="text-align:justify"><p><a href="https://github.com/friveramariani/genomic-data-science">Differential in Gene Expression between Fetal and Adult Brains</a>: In this genomic data science project, the goal was to evaluate differences in gene expression between fetal and adult brains. Transcriptome sequencing data (known as <a href="https://en.wikipedia.org/wiki/RNA-Seq">RNA-seq</a>) for this genomic data science project was sequenced on an Illumina platform from human post-mortem brains. The raw sequencing data and meta-data related to each brain sample was retrieved from a public database. The data was aligned, quality control performed on the alignments, and gene count levels of expression quantified. With the aid of exploratory and fitting statistical models,  patterns of gene expression were identified between fetus and adult brains. Findings from this genomic data science project provided insights into genes differentially expressed between fetus and adult brains that helps understand human development, and represents an example of reproducible research in genomic sciences.</p></div> 

- [GitHub repository](https://github.com/friveramariani/GenomicDataScience_FetalAdultBrain)
- [Find the PDF document here](https://www.researchgate.net/publication/311203295_Report_RNA-Seq_Data_Analysis_Worflow_to_Evaluate_Differential_Gene_Expression_between_Fetus_and_Adult_Brains_from_Publicly-Available_Data_as_a_Genomic_Data_Science_Demonstration_in_a_Upper_Division_Mi)
---

<div style="text-align:justify"><p><a href="https://www.researchgate.net/project/Applying-Data-Science-Approaches-in-Biological-Sciences-Classrooms?_esc=profileProjectCards&_sg=-XO68eEemQp3HNPS0KrOL2bjqDRIVkgGT7tgZU86f--Ied0fnhhIqTcwhregbtQPzGDLrd44MLrwDWCl0B9P9g.8bzPwU4Hw8H95Si2ifVxs30Y2D9G1NLXAWevaI20Ye1Iypn9hFDr6yxcGOegAuWT.eGebw20oCHpG6d88csNnnpo5djacTogpfurkn2o907J2sJAHKab4PoVXM_QRiQey&_viewIds%5B0%5D=Mfbk27SkHZVlqyAKuqvg1wx5&_viewIds%5B1%5D=6b1SML1ge7lBb56ZPFI5tTcU">Applying Data Science Approaches in Biological Sciences Classrooms</a>: Evidence-based decisions in the classroom are key for us as professor to be able to implement adjustments in students' learning, including in biological sciences courses. In this project, I'll be sharing data science approaches, projects, and peer-review publications, that could benefit making pedagogical adjustments in your classrooms.</p></div> 

***Recently-Published Reports***

- [Correct/Incorrect Ratio in Staining Techniques in Microbiology Lab](http://dx.doi.org/10.13140/RG.2.2.31839.66721)
- [Data Analysis in a General Education Biology Course after the First Summative Assessment](http://dx.doi.org/10.13140/RG.2.2.30379.85281)
- [An R function to generate proportions and logistic regression models of correct answers based on questions' topics, level of difficult, and question format](http://+n%20r%20function%20to%20generate%20proportions%20and%20logistic%20regression%20models%20of%20correct%20answers%20based%20on%20questions%27%20topic%2C%20levels%20of%20+ifficulty%2C%20and%20format./)
- [Text Mining R Code to Determine Most Frequent Words among Students' Hypotheses](http://dx.doi.org/10.13140/RG.2.2.16545.81763)
- [Data Analysis on a Survey Administered to Students on the 1st Day of Class in a General Education Biology Course](https://www.researchgate.net/project/Applying-Data-Science-Approaches-in-Biological-Sciences-Classrooms)

---

<div style="text-align:justify"><p><a href="http://rpubs.com/friveramariani/jhdscapstone">Shinny app to predict users' next word</a>: This projects describes  the pitch for a shiny app elaborated to predict next-words followings user's words input.</p></div>

- [Github repository](https://github.com/friveramariani/capstone-project)
 
---

Other biological and non-biological data sciences 

- [Find other Data Science Projects here](http://rpubs.com/friveramariani/dataproduct)
- [Find Biological and non-Biological Data Science codes here](https://github.com/friveramariani)

[**Click here to return to the Home Page**](http://www.friveram.com/)

<font size="1">&#169; Felix E. Rivera-Mariani, PhD 2017 <i>The contents of this website reflect the views of the author and does not represent the views of my institution.</i>.</font>